index.html

```diff
@@ -1,44 +1,226 @@
+                    </ul>
+                    <p class="primary-text">A smart assistant to make driving more safe, efficient, and fun by unlocking
+                        your most expensive computer: your car.</p>
+                    <ul class="categories">
+                        <li>html</li>
+                        <li class="ruby">Ruby on rails</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul><button class="button" id="4">See Project</button>
+                </div>
+            </div>
+        </div>
+        <section class="About_me" id="about_me">
+            <div class="about_me_full">
+                <h2 class="about_myself_h2">About Myself</h2>
+                <p class="about_myself_p"> Hello I’m a software developer! I can help you build a product , feature or
+                    website Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’thestiate to contact me. </p>
+                <p class="about_connect_2">LET'S CONNECT</p>
+                <ul class="the_media_icons_2">
+                    <li>
+                        <a href="#"><img class="social" src="images/header/twiter.png" alt="twitter" width="20" height="20"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img class="social" src="images/header/linkedin.png" alt="linkedin" width="20"
+                                height="20"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img class="social" src="images/header/m.png" alt="medium" width="20" height="20"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img class="social" src="images/header/github.png" alt="github" width="20" height="20"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img class="social" src="images/header/ok.png" alt="angellist" width="20" height="20"></a>
+                    </li>
+                </ul>
+                <button class="butn" type="submit"> Get my resume</button>
+            </div>
+            <div class="my_skills">
+                <ul class="my_skill_list">
+                    <li class="my_skill_one">
+                        <p>Languages</p>
+                        <a href="#"><img class="social" src="images/header/arrow.png" alt="down_cursor"></a>
+                    </li>
+                    <li><img src="images/header/javas.png"JavaScript"></li>
+                    <li><img src="images/header/ht.png" alt="Html"></li>
+                    <li><img src="images/header/cs.png" alt="Css"></li>
+                    <li class="my_skill_two">
+                        <p>Frameworks</p>
+                        <a href="#"><img class="social" src="images/header/desktop/Union.png" alt="L_cursor"></a>
```

Comment:
The same problem is occurring here kindly fix the src as I mentioned in the comment before. The images are not rendering because of the src property. :eyes: :eyes:

![image](https://user-images.githubusercontent.com/98363075/194715478-56cd0388-273a-43c5-86be-5dcd162d3aae.png)